src/views/general-partner-usual-residential-address-territory-choice.njk

```diff
@@ -45,7 +45,11 @@
         }) }}
               
         {{ govukButton({
-          text: i18n.buttons.continue
+          text: i18n.buttons.continue,
+          attributes: {
+            "id": "submit",
+            "onclick": "_paq.push(['trackEvent', ' general partner ura territory choice', 'selection', document.querySelector('input[name=parameter]:checked').nextElementSibling.innerHTML])"
```

Comment:
can the leading space in ' general partner ura territory choice' be removed or is it needed?